pegasus/forms/hoc_census.rb

```diff
@@ -0,0 +1,17 @@
+require pegasus_dir 'forms/hoc_signup_2017'
+
+class HocCensus2017 < HocSignup2017
+  def self.normalize(data)
+    result = super
+    # Once the change to split hoc signup and hoc census has been deployed we
+    # can move the census specific data extraction here. For now we want the
+    # old signup form to continue recording census data and we don't want to
+    # duplicate the data extraction here.
```

Comment:
Just so I understand what the plan is, is this correct?

1) We create two forms `HocSignup2017` and `HocCensus2017`.
2) Once the change is on production, new submissions will be recorded in the separate forms.
3) We'll migrate old census data in `HocSignup2017` to `HocCensus2017`.

Reply:
Yes, that's right. Although there isn't yet a specific plan around migrating the old data.